aws_topology/tests/cloudformation_infra/sqs-poll.py

```diff
@@ -0,0 +1,36 @@
+import sys
```

Comment:
Reason to add this file.

Reply:
@Eelviny Can we document that in Readme? How this file can be useful for testing and how to run it?

Reply:
Done!